### PR TITLE
Aligning 'Your COE' status pages with mockups

### PR DESCRIPTION
**Pull request by jerekshoe**
## Description
Align the status components for the `Eligibility` container with the mockups in uxpin

## Mockups
[available](https://preview.uxpin.com/65c0623a799c268173fe1a3cb4375f9ce00ad820#/pages/139512694/specification/sitemap)
[denied](https://preview.uxpin.com/65c0623a799c268173fe1a3cb4375f9ce00ad820#/pages/140537807/specification/sitemap)
[eligible](https://preview.uxpin.com/65c0623a799c268173fe1a3cb4375f9ce00ad820#/pages/139511607/specification/sitemap)
[ineligible](https://preview.uxpin.com/65c0623a799c268173fe1a3cb4375f9ce00ad820#/pages/140811192/specification/sitemap)
[pending](https://preview.uxpin.com/65c0623a799c268173fe1a3cb4375f9ce00ad820#/pages/139575305/specification/sitemap)

## Original issue(s)
department-of-veterans-affairs/va.gov-team#33332
department-of-veterans-affairs/va.gov-team#33448
department-of-veterans-affairs/va.gov-team#33554

## Acceptance criteria
- [ ] EligibilityApp Components match mockups

## Definition of done
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by cohnjesse — APPROVED** [review]
LGTM

**Review by jhonny218 — APPROVED** [review]
LGTM

File: src/applications/lgy/coe/containers/EligibilityApp.jsx [hunk 1]
```diff
@@ -39,33 +40,39 @@ const EligibilityApp = props => {
   let content;
 
   if (generateAutoCoeStatus === CALLSTATUS.idle || profileIsUpdating) {
-    content = <LoadingIndicator message="Loading application..." />;
+    content = <va-loading-indictator message="Loading application..." />;
```
Comment (k80bowman) [inline src/applications/lgy/coe/containers/EligibilityApp.jsx:43]: @jerekshoe so glad to see you using the web component loading indicator here (and below), but unfortunately it looks like you have a typo and the component is probably not loading properly for you.

Comment (jerekshoe) [reply src/applications/lgy/coe/containers/EligibilityApp.jsx:43]: Haha nice catch. I have another pull request out right now involving that file, I’ll be sure to update it there

**Pull request merged**